developerportal/apps/content/templates/content.html

```diff
@@ -6,18 +6,32 @@
 {% block body_class %}content{% endblock %}
 
 {% block content %}
-  <main>
+  <main class="{% if page.hero_image %}content-has-hero{% endif %}">
     <div class="base-grid">
       <section>
-        <h1>{{ page.title }}</h1>
-        {% for block in page.body %}
-          {% if block.block_type == 'image' %}
-            {% image block.value width-672 as rendition %}
-            {% include "atoms/image.html" with image=block.value rendition=rendition %}
-          {% else %}
-            {{ block }}
-          {% endif %}
-        {% endfor %}
+        {% if page.hero_image %}
+          {% image page.hero_image width-1024 as hero %}
+          <div class="content-hero">
+            <div class="content-hero-image" style="background-image: url('{{ hero.url }}')"></div>
+            <div class="content-hero-title">
+              <h1 class="highlighted">{{ page.title }}</h1>
+            </div>
+          </div>
+        {% else %}
+          <h1>{{ page.title }}</h1>
+        {% endif %}
+      </section>
+      <section class="article-content">
+        <article class="article-body">
+          {% for block in page.body %}
+            {% if block.block_type == 'image' %}
+              {% image block.value width-672 as rendition %}
```

Comment:
Sorry, just noticed perhaps this image size should be updated? I think maybe this is why they're having image issues with this particular page type.

Reply:
Content is now centred so this rendition size is now correct.